### ISSUE TITLE
error with gmt movie: FFmpeg not found

### ISSUE THREAD

**Issue by mshastings18** [gmtsar/gmtsar#30]
I installed gmt version 6.1 on Friday and the first test case worked fine, however, when I ran the second one I got an error with gmt movie telling me that ffmpeg is not installed. I uninstalled and purged it from my machine and then tried to reinstall it freshly but that did not fix the problem. You can also see that if I try to just run ffmpeg from the command line I get an error with a shared library but I am not sure if that is the problem or not since gmt seems to think it is not installed. 

I installed gmt using anaconda and I am currently running on Ubuntu 18.04. Any help is appreciated!

Cheers,

Mitch

![ffmpeg_problem](https://user-images.githubusercontent.com/31482314/87878848-e61a4900-c9b4-11ea-8f45-596c38cbbb44.png)

**Comment by dsandwell** [comment]
Hi Mitch,

You may need to install ffmpeg with apt-get.

Also GMTSAR will not install with a GMT installed with anaconda.  The ./configure file cannot find the GMT libraries.

So when you get GMT6.1 working and if you want GMTSAR you will also need to install GMT using apt-get following the UBUNTU 16 instructions at the WIKI

https://github.com/gmtsar/gmtsar/wiki

Then you can set your path so it looks for the GMT anaconda first.

Sorry about this confusion.  We will try to get the conda version to link after the short course.

David


> On Jul 19, 2020, at 8:45 AM, mshastings18 <notifications@github.com> wrote:
> 
> 
> I installed gmt version 6.1 on Friday and the first test case worked fine, however, when I ran the second one I got an error with gmt movie telling me that ffmpeg is not installed. I uninstalled and purged it from my machine and then tried to reinstall it freshly but that did not fix the problem. You can also see that if I try to just run ffmpeg from the command line I get an error with a shared library but I am not sure if that is the problem or not since gmt seems to think it is not installed.
> 
> I installed gmt using anaconda and I am currently running on Ubuntu 18.04. Any help is appreciated!
> 
> Cheers,
> 
> Mitch
> 
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> 

David T. Sandwell
IGPP 0225
Scripps Inst. of Oceanography
La Jolla, CA
92093-0225

Ph. 858 534-7109
Cell 858 663-9426
http://topex.ucsd.edu

Express Mail Address:
8795 Biological Grade
Room 1102
La Jolla, CA 92037

**Comment by mshastings18** [comment]
Hi David, 

Thanks for the quick reply! I have attempted to install ffmpeg with apt, apt-get, and snap but none seem to work and gmt cannot find it. I removed and purged ffmpeg and tried again with apt-get (though I think it is an alias for apt at this point, right?), but to no avail. I have tested the gmt 6.1 install with some of my old gmt codes to make sure the static images and PostScript output works fine and it seems to be working beautifully (really like the new verbose output!). 

Luckily I was just about to start fiddling with installing GMTSAR, but good thing I read this first! 

Thanks,

Mitch
